custom/ewsghana/templates/ewsghana/partials/location_async.html

```diff
@@ -0,0 +1,32 @@
+{% load hq_shared_tags %}
```

Comment:
can you clarify what the difference is between this and https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/filters/location_async.html is? any way for this one to extend that one?